pylint/checkers/refactoring.py

```diff
@@ -525,10 +529,72 @@ def _check_consider_using_in(self, node):
 
         self.add_message('consider-using-in', node=node, args=(suggestion,))
 
-    @utils.check_messages('consider-merging-isinstance', 'consider-using-in')
+    def check_chained_comparison(self, node):
+        """Check if there is any chained comparison in the expression.
+        If it exists, give message to simplify it."""
+        if node.op != 'and':
+            return
+
+        total_compares = len(node.values)
+        if total_compares < 2:
+            return
+
+        # all node values must be compare
```

Comment:
This can also be `if not all(isinstance(value, astroid.Compare) for value in node.values): `